contracts/tasks/deploy/maci/02-gatekeepers.ts

```diff
@@ -110,5 +118,39 @@ deployment.deployTask("full:deploy-gatekeepers", "Deploy gatekeepers").then((tas
         network: hre.network.name,
       });
     }
+
+    if (!skipDeployZupassGatekeeper) {
+      const eventId = deployment.getDeployConfigField<string>(EContracts.ZupassGatekeeper, "eventId", true);
+      const validEventId = uuidToBigInt(eventId);
+      const validSigner1 = deployment.getDeployConfigField<string>(EContracts.ZupassGatekeeper, "signer1", true);
+      const validSigner2 = deployment.getDeployConfigField<string>(EContracts.ZupassGatekeeper, "signer2", true);
+      let verifier: string | undefined = deployment.getDeployConfigField<string>(
+        EContracts.ZupassGatekeeper,
+        "zupassVerifier",
+      );
+      if (!verifier) {
+        const verifierContract = await deployment.deployContract({
+          name: EContracts.Groth16Verifier,
```

Comment:
let's change the contract name to be ZupassVerifier (or ZupassGroth16Verifier)

Reply:
Got it! thought it was only for the config variable and not confuse coordinators